missions/31_pony.txt

```diff
@@ -0,0 +1,158 @@
+quest_for_harmony_south_america = {
```

Comment:
Here's the mission from your own pony.txt mission file. Just consolidating everything into one.